websites/W/Warriorcats/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "202740603790819328",
+		"name": "aimslut"
+	},
+	"service": "Warriorcats",
+	"description": {
+		"en": "meow"
```

Comment:
This is not the official description of the website. Add the official description and/or translate if needed.

Reply:
Oh yeah. That was just placeholer. I forgot that :D